org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -1203,7 +1178,89 @@ else if (obj instanceof PrintData)
 		return true;
 	}	//	createHTML
 
-
+	/**
+	 * Get record identifier string
+	 * @param mTable
+	 * @param tableName
+	 * @param recordID
+	 * @return String identifier
+	 */
+	private String getIdentifier(MTable mTable, String tableName, int recordID) {
+		ArrayList<MColumn> list = new ArrayList<MColumn>();
+		// get translation table - null if not exists
+		MTable mTableTrl = MTable.get(getCtx(), tableName+"_Trl");
+		String tableNameTrl = "";
+		// get report language
+		String reportLang = getLanguageID() > 0 ? new MLanguage(getCtx(), getLanguageID(), null).getAD_Language() : Language.getLoginLanguage().getAD_Language();
+		
+		// use Trl table or base table
+		boolean isTrl = !Env.isBaseLanguage(Language.getLanguage(reportLang), tableName);
+		
+		if(isTrl && mTableTrl != null && getLanguageID() > 0)
```

Comment:
with changes at line 1194, the "getLanguageID() > 0" condition should be removed.

Reply:
I see, thanks, fixed